doc/tutorials/tutorials.rst

```diff
@@ -33,3 +33,8 @@ Get details about Cox datafit equations.
 -----------------------------------------------------------------
 
 Mathematical details about the group Lasso, in particular with nonnegativity constraints.
+
+:ref:`Mathematics Behind L1 Regularization and Fermat's Rule <fermat_rule_reg>`
```

Comment:
I'd rename this critical régularisation strength above which solution is 0 (less maths, more to the point)